src/InputGroup/index.js

```diff
@@ -138,10 +138,6 @@ class InputGroup extends React.PureComponent<Props, State> {
     this.isFilled();
   }
 
-  componentDidUpdate() {
-    this.isFilled();
```

Comment:
Why this removal ?

Reply:
Tests were failing, I just deleted and forgot about it.
With upgrading enzyme to the new version, tests will fail for this component. `componentDidUpdate` calling itself in an infinite loop, we need to fit it in separate PR.

Reply:
```
RangeError: Maximum call stack size exceeded
      144 | 
      145 |   isFilled = () =>
   > 146 |     this.setState({
             |          ^
      147 |       filled: !React.Children.map(
      148 |         this.props.children,
      149 |         child => child.props.value !== undefined && child.props.value !== "",
```